setup.py

```diff
@@ -18,11 +18,11 @@ def readme():
 
 
 setup(name='pycompwa',
-      version='0.0.1',
+      version='0.1.00',
```

Comment:
is that supposed to be 0.1.00, not 0.1.0?

Reply:
I guess thats the same version on pypi.